subsys/bluetooth/host/id.c

```diff
@@ -974,6 +974,11 @@ void find_rl_conflict(struct bt_keys *resident, void *user_data)
 			bt_irk_eq(&conflict->candidate->irk, &resident->irk));
 
 	if (addr_conflict || irk_conflict) {
+		LOG_DBG("Resident : addr %s and IRK %s", bt_addr_le_str(&resident->addr),
```

Comment:
```suggestion
		LOG_DBG("Resident: addr %s and IRK %s", bt_addr_le_str(&resident->addr),
```

Reply:
The reason for the space was to align it width-wise with `Candidate:`